mantle/platform/api/gcloud/compute.go

```diff
@@ -120,14 +162,27 @@ func (a *API) mkinstance(userdata, name string, keys []*agent.Key, useServiceAcc
 			OnHostMaintenance: "TERMINATE",
 		}
 	}
-	return instance
-
+	// attach aditional disk
+	if len(opts.AdditionalDisks) > 0 {
+		for _, spec := range opts.AdditionalDisks {
```

Comment:
Completely minor nit, but is the `len` check here needed?  Won't the loop be a no-op if the length is zero anyways?